xbmc/video/VideoInfoScanner.cpp

```diff
@@ -818,6 +886,72 @@ namespace VIDEO
     return INFO_NOT_FOUND;
   }
 
+  bool CVideoInfoScanner::UpdateSetInfo(CFileItem* pItem)
```

Comment:
Why is this a raw pointer? Could this be changed to `CFileItem&` or even better `const CFileItem&`?
Please remove the p prefix. Coding guidelines.

Reply:
Only because that's what pItem is in the calling code. I've changed it.